### DISCUSSION TITLE
Improve visibility of "minor" grid lines

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#29721] [Ideas] [answered]
Currently the "minor" grid lines are barely visible on a 4k monitor at normal distances:
![1](https://github.com/user-attachments/assets/1c4d143a-925c-4950-8b6a-da970f4d4bc5)
![2](https://github.com/user-attachments/assets/b8473e7b-ed85-42cf-807a-ccc2d56544ec)
It would be nice if they can be made more visible, e.g. by higher thickness or a brighter color.
It should be an option though imo, to avoid them being annoying when the grid is not used (or alternatively, add a way to hide the grid completely, like those purposed in https://github.com/ppy/osu/discussions/29713)

**Comment by peppy** [comment, marked as answer]
Right now, the line thickness is fixed regardless of display resolution. I think this is incorrect. It should have a minimum thickness, but get thicker when the resolution / DPI is high enough to warrant it.

https://github.com/ppy/osu/issues/30072
